Assets/HoloToolkit/Sharing/Tests/ImportExportAnchorManager.cs

```diff
@@ -116,18 +116,19 @@ public bool AnchorEstablished
     /// </summary>
     RoomManagerAdapter roomManagerCallbacks;
 
-    void Start()
+    void Awake()
```

Comment:
Keep in mind that the Singleton pattern is being updated in part to the new input modules